src/registrar/templates/home.html

```diff
@@ -121,13 +119,13 @@ <h2>Active domain requests</h2>
   </section>
 
   <section class="section--outlined tablet:grid-col-11 desktop:grid-col-10">
-    <h2>Archived domains</h2>
-    <p>You don't have any archived domains</p> 
+    <h2>Archive</h2>
+    <p>You don't have anything in the archive yet.</p> 
   </section>
 
   <section class="tablet:grid-col-11 desktop:grid-col-10">
     <h2 class="padding-top-1 mobile-lg:padding-top-3"> Export domains</h2>
-    <p>Download a list of your domains and their statuses as a csv file.</p>
+    <p>If you would like to analyze your list of domains further, you can download your list of domains and their statuses as a csv file.</p>
```

Comment:
@Katherine-Osos Can you take a look at this? I never looked at the original copy, but it had some grammatical and spelling errors. Zander has passed through it and cleaned it up, but something about the sentence structure feels off to me. Maybe it's the use of "your" so much?

Reply:
@gabydisarli Personally, I found the original text here to be a bit more concise: "Download your list of domains and their statuses as a csv file.", but if we're keeping with the spirit of the Figma doc maybe we can reword it as so? 
`For further analysis, you can download your list of domains and their statuses as a csv file.`
(Maybe we could also say 'for your convenience' instead of 'for further analysis' here)

Reply:
Was this the original verbiage?: "Download a list of your domains and their statuses as a csv file."

If so, I like that better. 

Also, it looks like this content is going to be temporarily removed based on [this slack thread ](https://cisa-corp.slack.com/archives/C05BDEA3C11/p1691455212627079)and the new ticket created: #860

Reply:
Actually, since this is getting removed in https://github.com/cisagov/getgov/issues/860 let's shelve this conversation.